tests/browser/browser_test.js

```diff
@@ -18,7 +18,7 @@ const ABOUT_BLANK = "about:blank";
 const URLBAR_PROVIDER_NAME = "ProviderDynamicQuickSuggest";
 
 const ATTRIBUTION_URL =
-  "http://mochi.test:8888/browser/testing/extensions/browser/qs_attribution.sjs";
+  "http://mochi.test:8888/browser/testing/extensions/browser/qs_attribution.sjs?";
```

Comment:
This needed added due to a change in PartnerLinkAttribution that appends the campaignID expecting that there is already a queryString